docker-compose.yml

```diff
@@ -30,24 +30,29 @@ services:
       - isard-app
 
   isard-hypervisor:
+    environment:
+      - HTML5_WEBSOCKETS=${HTML5_WEBSOCKETS}
     container_name: isard-hypervisor
     volumes:
-      - "/opt/isard/sshkeys:/root/.ssh"
+      - "/opt/isard/sshkeys/$HOSTNAME:/root/.ssh"
       - "/opt/isard:/isard"
       - "/opt/isard/certs/default:/etc/pki/libvirt-spice"
       - "/etc/localtime:/etc/localtime:ro"
     ports:
-      - "5900-5999:5900-5999"
-      - "55900-55999:55900-55999"
+      - "2022:22"
+      - "5900-6199:5900-6199"
+      - "6400-6699:6400-6699"
     networks:
       - isard_network
+#      - default
+#      - internet-vms
```

Comment:
This comments could be deleted, since they're not used for anything

Reply:
They are needed if you want to use vlans